app/routes/_index.tsx

```diff
@@ -44,7 +54,13 @@ export default function Index() {
             Remix Docs
           </a>
         </li>
+        <Button onClick={handleCreateRole}>Create Role</Button>
       </ul>
+
+      <SuccessModal
+        isOpen={isSuccessModalOpen}
```

Comment:
Why is this in the index route though?

Please remove it and ensure it follows the flow of the UI design.

Reply:
do i remove the entirety of the component from the _index.tsx. but how will I test the functionality of the component?